### DISCUSSION TITLE
How long will it take to get benefits after getting green verified tag?

### DISCUSSION THREAD

**Discussion by PrabhasKmrGH** [community/community#75488] [GitHub Education] [answered]
### Hi everyone,

**Greetings everyone**

First of all I apologize if this topic has been discussed before; I'm reaching out here to seek information about the expected timeline for benefits approval. I submitted my application on _**November 11, 2023**_, and I received an approval notification within minutes - indicated by the green tag, but the benefits haven't arrived yet.

<img width="1436" alt="Screenshot 2023-11-14 at 3 28 28 PM" src="https://github.com/community/community/assets/85011883/7d3fe289-52a1-485f-ad35-5695484fb6c3">

I've heard that a you need **_purple tag_** you get purple tag instead of green tag of 'Approved'. I want to know more about this.

Additionally, I would appreciate guidance on when it might be appropriate to consider reapplying, if necessary.

Thank you in advance for your assistance.

Regards,

Prabhas Kumar

> Note: This message might have been generated or modified using AI.

**Comment by AidenJamesYt** [comment]
you will typically receive your benefits within 1-2 weeks, but sometimes it may take longer. if you have not received your benefits within 2 weeks, you should contact github directly to inquire about the status of your benefits.

**Reply by PrabhasKmrGH** [reply]
I'm not sure whether you accelerated it, or it just happened on its own, but a few hours ago, I was able to get Copilot without realizing it (I thought it was VS Code autocompletion). I just checked five minutes ago, and I've got all the benefits. Thanks anyway.

**Comment by PrabhasKmrGH** [comment, marked as answer]
I'm not sure whether anyone accelerated it, or it just happened on its own, but a few hours ago, I was able to get Copilot without realizing it (I thought it was VS Code autocompletion). I just checked five minutes ago, and I've got all the benefits. Thanks anyway.

**Comment by RemasteredGod** [comment]
go to this site and here you can see all the benefits that are provided in the students pack **https://education.github.com/globalcampus/student**
